### ISSUE TITLE
Installing packages returns warning with unocss

### ISSUE THREAD

**Issue by Rednas83** [formkit/formkit#1295]
### Reproduction

Not applicable

### Describe the bug

After updating/installing formkit I am getting a peer dependency warning with unocss.
![image](https://github.com/formkit/formkit/assets/25219229/cba739bd-77b0-45c7-a529-28a1ea9dcb1a)
**Package.json**
```
    "@formkit/nuxt": "^1.6.1",
    "@formkit/pro": "^0.122.15",
    "@formkit/themes": "^1.6.1",
    "@formkit/addons": "^1.6.1",
    "@formkit/icons": "^1.6.1",   
```
I mostly ignore it, but it's a bit annoying because formkit is as for as I know not using any unocss.

### Environment

![image](https://github.com/formkit/formkit/assets/25219229/6c3926a3-f461-4d63-9c26-0bdc3fe3e435)

**Comment by Archetipo95** [comment]
I think this will be fixed in the next release #1281

**Comment by justin-schroeder** [comment]
This should be fixed in 1.6.2
